packages/material-ui/src/Dialog/Dialog.test.js

```diff
@@ -181,6 +181,46 @@ describe('<Dialog />', () => {
         'should not fire the onBackdropClick callback',
       );
     });
+
+    it(
+      'should not close if the dialog receives the mousedown and the ' +
+        'backdrop receives the mouseup',
+      () => {
+        const onBackdropClick = spy();
+        wrapper.setProps({ onBackdropClick });
+
+        const backdrop = wrapper.find('div');
+        const content = wrapper.find('withStyles(Paper)').childAt(0);
```

Comment:
```suggestion
        const content = wrapper.find('[role="dialog"]');
```
and then add that role to the paper. Improves a11y as a nice side-effect. In general we try to move away from `childAt` usage. It's way to brittle WRT to context, forwardRef etc.